google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/scrape/config.py

```diff
@@ -14,55 +14,153 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+import json
+import os
 import yaml
 
-from .config_constants import ENRICH_METADATA_OPTION
-from .config_constants import REFRESH_OPTION
+from google.datacatalog_connectors.rdbms.scrape import config_constants
 
 
 class Config:
 
-    def __init__(self, config_path):
-        self._config_path = config_path
-        self._conf_content = self._read_config_file()
+    def __init__(self, user_config_path, connector_config_path):
+        self._user_config_path = user_config_path
+        self._connector_config_path = connector_config_path
+        self._conf_content = self.__read_yaml_file(self._user_config_path)
         self.refresh_metadata_tables = False
         self.scrape_optional_metadata = False
-        self._determine_scraping_steps()
-
-    def _read_config_file(self):
-        with open(self._config_path, 'r') as f:
-            conf = yaml.load(f, Loader=yaml.FullLoader)
-        return conf or dict()
-
-    def _determine_scraping_steps(self):
-        if self._conf_content.get(REFRESH_OPTION) is not None:
-            self.refresh_metadata_tables = self._conf_content[REFRESH_OPTION]
-        options = self.get_chosen_metadata_options()
-
-        if len(options):
-            self.scrape_optional_metadata = True
+        self.__determine_scraping_steps()
 
     def get_chosen_metadata_options(self):
         '''
-        From the config contents, retrieve options that user has marked as true
+        From the config contents,
+        retrieve options that user has marked as true
         '''
         options = [
             # TODO put the scrape_options in a parent
             #  key in the user def config
             option
             for option, choice in self._conf_content.items()
-            if choice and
-            (option != REFRESH_OPTION and option != ENRICH_METADATA_OPTION)
+            if choice and (option != config_constants.REFRESH_OPTION and
+                           option != config_constants.ENRICH_METADATA_OPTION)
         ]
         return options
 
-    def get_content_dict_attribute(self, attribute_name):
+    def get_sql_objects_config(self):
+        parsed_config = {}
+
+        sql_objects = self._conf_content.get(config_constants.SQL_OBJECTS_KEY)
+
+        if sql_objects:
```

Comment:
```suggestion
        if not sql_objects:
            return parsed_config
```
Same suggestion to avoid nesting code...